src/common/syncjournaldb.cpp

```diff
@@ -851,8 +855,8 @@ bool SyncJournalDb::updateMetadataTableStructure()
 
     addColumn(QStringLiteral("isLivePhoto"), QStringLiteral("INTEGER"));
     addColumn(QStringLiteral("livePhotoFile"), QStringLiteral("TEXT"));
-    addColumn(QStringLiteral("quotaBytesUsed"), QStringLiteral("BIGINT"));
-    addColumn(QStringLiteral("quotaBytesAvailable"), QStringLiteral("BIGINT"));
+    addColumn(QStringLiteral("quotaBytesUsed"), QStringLiteral("BIGINT"), false, QStringLiteral("DEFAULT -1 NOT NULL"));
+    addColumn(QStringLiteral("quotaBytesAvailable"), QStringLiteral("BIGINT"), false, QStringLiteral("DEFAULT -1 NOT NULL"));
```

Comment:
this contradicts with the lines 75-76 which do test for `NULL` values
I would advice to have one way or the other to decrease complexity
so either you add `NOT NULL` for the column or you test for `NULL` values